README.md

```diff
@@ -1,6 +1,14 @@
 # EveryPolitician Pull Request summary
 
-Produces a summary of an EveryPolitician data pull request like [this one](https://github.com/everypolitician/everypolitician-data/pull/16984#issuecomment-246282991).
+This is the code the EverPolitician bot uses to inspect data pull requests (specifically those are on the
+[everypolitician-data](https://github.com/everypolitician/everypolitician-data) repo)
+and add helpful summaries to them. The bot writes the summary as a comments on the Pull Request
```

Comment:
Yikes! Fixed in https://github.com/everypolitician/everypolitician-pull_request/pull/8/commits/cac2fffbca396b9b45952f173756565e1f04dc77.